data-plane/tests/pom.xml

```diff
@@ -105,6 +105,12 @@
       <version>${kafka.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.awaitility</groupId>
+      <artifactId>awaitility</artifactId>
+      <version>3.0.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Instead of declaring the version here and in `dispatcher/pom.xml`, can we use a similar technique to `cloudevents-kafka`:

Dependency declaration:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/17e3199118d2a9d47c2bb9319c4b392446503e99/data-plane/pom.xml#L155-L165

Usage:
https://github.com/knative-sandbox/eventing-kafka-broker/blob/17e3199118d2a9d47c2bb9319c4b392446503e99/data-plane/dispatcher/pom.xml#L66-L69

Reply:
Also, can we use the latest version `4.0.3`? https://mvnrepository.com/artifact/org.awaitility/awaitility/4.0.3

Reply:
Yes, we can.  :D 👍